Adding pvcaptest package

bt- opened the pull request:
<!--
Thank you very much for putting in this recipe PR!

This repository is very active, so if you need help with
a PR or once it's ready for review, please let the right people know.
There are language-specific teams for reviewing recipes.

Currently available teams are:
- python `@conda-forge/help-python`
- python/c hybrid `@conda-forge/help-python-c`
- r `@conda-forge/help-r`
- java `@conda-forge/help-java`
- nodejs `@conda-forge/help-nodejs`
- c/c++ `@conda-forge/help-c-cpp`
- perl `@conda-forge/help-perl`
- Julia `@conda-forge/help-julia`
- ruby `@conda-forge/help-ruby`

If your PR doesn't fall into those categories please contact
the full review team `@conda-forge/staged-recipes`.

Due to GitHub limitations first time contributors to conda-forge are unable
to ping these teams. You can [ping the team](https://conda-forge.org/docs/maintainer/infrastructure.html#conda-forge-admin-please-ping-team) using a special command in
a comment on the PR to get the attention of the `staged-recipes` team. You can
also consider asking on our [Gitter channel](https://gitter.im/conda-forge/conda-forge.github.io)
or on our [Keybase chat](https://keybase.io/team/condaforge.chat)
if your recipe isn't reviewed promptly.
-->

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.

Yes, I am submitting the pull request and the only maintainer listed.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipes/pvcaptest```) and found it was in an excellent condition.

marcelotrevisani commented on recipes/pvcaptest/meta.yaml:
```diff
@@ -0,0 +1,56 @@
+
+requirements:
+  host:
+    - bokeh >=1
+    - matplotlib >=2
+    - numpy >=1.13.0
+    - pandas >=0.24
+    - pip
+    - setuptools
+    - python >=3.5
+    - python-dateutil >=2.5
+    - scikit-learn >=0.19
+    - statsmodels >=0.8
+    - holoviews >=1.11
+    - pvlib-python >=0.7
```
I think here you just need ``python`` and ``pip`` to package this project
```suggestion
    - pip
    - python >=3.5
```

bt- replied:
I removed the other packages from host section.

marcelotrevisani commented on recipes/pvcaptest/meta.yaml:
```diff
@@ -0,0 +1,56 @@
+
+  run:
+    - bokeh >=1
+    - matplotlib >=2
```
```suggestion
    - matplotlib-base >=2
```

bt- replied:
I made this change to matplotlib-base.  What is the difference between these two?

marcelotrevisani replied:
If you do not need the ``qt`` backend just ``matplotlib-base`` is sufficient 
``matplotlib`` will pull ``pyqt`` and the ``qt`` stack, which are a quite large dependency

marcelotrevisani commented on recipes/pvcaptest/meta.yaml:
```diff
@@ -0,0 +1,56 @@
+
+test:
+  imports:
+    - captest
```
Can we try to add ``pip check`` here please?
```suggestion
    - captest
  requires:
    - pip
  commands:
    - pip check
```

bt- replied:
I added the pip check test.  This look to have passed when I built it locally. before committing these revisions.

marcelotrevisani commented on recipes/pvcaptest/LICENSE.txt:
```diff
@@ -0,0 +1,21 @@
+MIT License
```
It seems the license is already being packaged on pypi, you can remove this file then

bt- replied:
I removed the duplicate license file.

marcelotrevisani commented:
If you want to, next time you might give a go to [grayskull](https://github.com/marcelotrevisani/grayskull to generate recipes for packages which are on ``pypi``

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipes/pvcaptest```) and found some lint.

Here's what I've got...


For **recipes/pvcaptest**:

 * license_file entry is missing, but is required.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipes/pvcaptest```) and found it was in an excellent condition.

bt- commented:
@marcelotrevisani, thank you for the review.  I think I've addressed all your comments.  I tried using grayskull through your website, which seemed to work great.  I'll keep it mind if I need to create another recipe for a package on pypi.

marcelotrevisani commented:
Nice! Thanks for adding this project to ``conda-forge``! :tada:  :tada:  :tada:  :tada:
